app/views/passengers/edit.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1> Edit Passenger</h1>
+
+<% if @passenger.errors.any? %>
+  <ul class="errors">
+    <% @passenger.errors.each do |column, message| %>
+      <li>
+        <strong><%= column.capitalize %></strong> <%= message %>
+      </li>
+    <% end %>
+  </ul>
+<% end %>
+
+<%= form_for @passenger do |f| %>
+  <%= f.label :name %>
+  <%= f.text_field :name %>
```

Comment:
You have the same form for both `edit` and `new`. Could you DRY this up with a view partial?